tools/buildsteps/windows/vswhere.bat

```diff
@@ -44,15 +44,13 @@ IF "%vcstore%"=="store" (
 
 SET vswhere="%builddeps%\%toolsdir%\tools\vswhere\vswhere.exe"
 
-FOR /f "usebackq tokens=1* delims=" %%i in (`%vswhere% -latest -property installationPath`) do (
-  IF EXIST "%%i\VC\Auxiliary\Build\vcvarsall.bat" (
-    SET vcvars="%%i\VC\Auxiliary\Build\vcvarsall.bat"
+IF EXIST "C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Auxiliary\Build\vcvarsall.bat" (
```

Comment:
Can you remove the changes to this file? It doesn't work for vs2017 pro and not sure it works correctly for vs2019 either

Reply:
Oops! I did not mean to commit that!
I have both VS 2017 and 2019 installed and was having trouble getting this script to choose 2017 so I forced it to do so.
Reverted!